analyses/cell-type-wilms-tumor-06/scripts/06_infercnv.R

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env Rscript
+
+# Run `infercnv` for one sample with or without a healthy reference
+# infercnv
+#
+# USAGE:
+# Rscript 06_infercnv.R \
+#   --sample_id SCPCS000194
+#   --ncore 16
```

Comment:
This is not an argument; this is left over from your other script. Please update this usage comment to match this script's arguments